platform/view/services/storage/driver/sql/postgres/sql_test.go

```diff
@@ -49,3 +55,86 @@ func TestPostgres(t *testing.T) {
 		return p.(*KeyValueStore).KeyValueStore
 	})
 }
+
+func verifyExtendedLoggingInCommand(t *testing.T, config *ContainerConfig, expectExtendedLogging bool) {
+	cli, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+	require.NoError(t, err)
+
+	ctx := context.Background()
+	containers, err := cli.ContainerList(ctx, container.ListOptions{})
+	require.NoError(t, err)
+
+	// Find our container
+	var foundContainer *types.Container
+	for _, c := range containers {
+		for _, name := range c.Names {
+			if name == "/"+config.Container {
+				foundContainer = &c
+				break
+			}
+		}
+	}
+	require.NotNil(t, foundContainer, "Container not found")
+
+	// Inspect the container to get the command
+	inspect, err := cli.ContainerInspect(ctx, foundContainer.ID)
+	require.NoError(t, err)
+
+	// Check extended logging parameters based on expectation
+	cmd := inspect.Config.Cmd
+	if expectExtendedLogging {
+		assert.Contains(t, cmd, "log_destination=stderr")
```

Comment:
I am wondering if there is a particular reason to use `assert` over `require` here. The difference is that `assert` does not call `FailNow`.